### PR TITLE
wr-1109 Career Profile Education styling and EducationPlannerBC Static Content

### PR DESCRIPTION
**Pull request by Mirzero**

### CODE REVIEW

File: src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig [hunk 1]
```diff
@@ -398,6 +398,22 @@
...
+              {% for program in viewmodel['tab_sections']['education']['education_programs_list'] %}
+                {{ program.0['#markup']|raw }}
+              {% endfor %}
+
+              <div class="card-profile">
+                <div class="card-profile__header">
+                  <img src="/{{ directory }}/assets/images/logo-educationplannerbc.png" alt="EducationPlannerBC Logo" />
+                </div>
+                <div class="card-profile__content-left">
+                  <div class="card-profile__content-item">Find out more information about programs offered specifically for this career.</div>
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig:410]: is there a link to Education Planner BC with the specific NOC ?

Comment (rjkuyvenhoven) [reply src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig:410]: The Education Program extra field provides a link that includes the noc number. However, I think is probably better to remove that link from the extra field and construct the appropriate link in this template.

The noc number should be available from the content. 

This is the link format: https://www.educationplannerbc.ca/search/noc/[noc#]

I will remove the link code from the Education Programs extra field

File: src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig [hunk 2]
```diff
@@ -398,6 +398,22 @@
               <h2 class="profile-content-section__header">{{ viewmodel['tab_sections']['education']['education_programs_title'] }}</h2>
               <div>{{ viewmodel['tab_sections']['education']['education_programs_list']|raw }}</div>
 
+              {% for program in viewmodel['tab_sections']['education']['education_programs_list'] %}
+                {{ program.0['#markup']|raw }}
+              {% endfor %}
+
+              <div class="card-profile">
+                <div class="card-profile__header">
+                  <img src="/{{ directory }}/assets/images/logo-educationplannerbc.png" alt="EducationPlannerBC Logo" />
```
Comment (zetagraph) [inline src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig:407]: Would be nice to use an SVG instead but assuming we can't source those?

File: src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig (hunk 1 above)
Comment (Mirzero) [reply src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig:410]: I've made this change.

File: src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig (hunk 2 above)
Comment (Mirzero) [reply src/web/themes/custom/workbc/templates/content/node--career-profile--full.html.twig:407]: I agree, but an SVG wasn't provided. We could go back to Michaela/Client/etc. to ask for one, but that's probably not priority at the moment.

**Review by rjkuyvenhoven — APPROVED** [review]
LGTM

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks

**Pull request merged**